UPGRADING.md

```diff
@@ -0,0 +1,22 @@
+UPGRADING
+=========
+## 1.x &rarr; 2.x
+
+### PHP 8.0 required
```

Comment:
Should we maybe confirm with OPS that we do not have a server that cannot meet this requirement

Reply:
I'm thinking that if we have servers with projects still using older PHP versions, the projects should declare the PHP requirements in composer.json correctly, so dependencies are not upgraded to this version prematurely.

Reply:
Ok